### PR TITLE
update 19/11

### PR DESCRIPTION
**Pull request by manhbapper19**
merge login into existing feature

### CODE REVIEW

File: build/classes/view/HomeFrm.form [hunk 1]
```diff
@@ -93,7 +93,7 @@
                       <Component id="jPanel4" min="-2" max="-2" attributes="0"/>
                       <EmptySpace type="unrelated" max="-2" attributes="0"/>
                       <Component id="jpnSanPham" min="-2" max="-2" attributes="0"/>
-                      <EmptySpace type="unrelated" max="-2" attributes="0"/>
+                      <EmptySpace min="-2" pref="14" max="-2" attributes="0"/>
```
Comment (manhbapper19) [inline build/classes/view/HomeFrm.form:96]: accept this

**Review by manhbapper19 — COMMENTED** [review]
fine

**Comment by Emiliahh** [comment]
con cac

**Review by Emiliahh — CHANGES_REQUESTED** [review]
ahh

**Pull request merged**